azurelinuxagent/distro/suse/rdma.py

```diff
@@ -35,10 +35,13 @@ def install_driver(self):
             logger.error(error_msg)
             return
         zypper_install = 'zypper -n in'
+        zypper_remove = 'zypper -n rm'
         zypper_search = 'zypper se -s'
```

Comment:
maybe extract these `zypper` commands just below `class RDMAHandler` (and have trailing `%s` so we can format easier)?